distrobox-create

```diff
@@ -166,9 +166,20 @@ generate_command() {
 
 # First, check if the image exists in the host
 if ! podman image exists "${container_image}"; then
-	echo "Image not found, run this command first:"
```

Comment:
can you run

`shfmt -s <file-name>`

on the files? It unifies the indentation with the rest of the code this way

to install it `go install mvdan.cc/sh/v3/cmd/shfmt@latest`